compiler/executable/match_/planner/match_executable.rs

```diff
@@ -192,8 +192,8 @@ impl Display for IntersectionStep {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         write!(
             f,
-            "inputs={:?}, output_size={}, sort_by={}",
-            &self.input_variables, self.output_width, self.sort_variable
+            "bound_vars={:?}, output_size={}, sort_by={}",
```

Comment:
"Bound" variables is more accurate because these are only the variables that are mentioned in this step/instruction AND inputs (eg. bound)